game/scripts/vscripts/components/duels/duel.lua

```diff
@@ -0,0 +1,156 @@
+Duel = class({})
+
+function Duel:Init (teams, spawns, zone)

```

Comment:
Is Init not recognised by class() as its constructor? I actually just realised that classes aren't a vanilla lua thing and now I'm wondering where they're implemented :-)